app/subscriber/src/features/my-reports/edit/settings/styled/ReportEditSubscribersForm.tsx

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components';
+
+export const ReportEditSubscribersForm = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin: 1rem;
+  gap: 1rem;
+  .select-column {
+    width: 5rem !important;
```

Comment:
Is there a way we can do this without using `!important`?  Generally this only needed when we're using a 3rd party component that wasn't written well.